# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/metamail/60_metamail-2.7-fixawk.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- mm2.7/src/bin/sun2mime.fixawk	Thu Feb  1 12:28:33 2001
+++ mm2.7/src/bin/sun2mime	Thu Feb  1 12:28:55 2001
@@ -20,7 +20,7 @@
     rm -rf $TMPFILE
 fi
 
-/bin/nawk '
+/bin/gawk '
   BEGIN { printf("Content-type: multipart/mixed; boundary=--------\n\n")
           RS=""; FS="\n"; mode="HEADER" }
   mode == "HEADER" {
